cves/kernel/CVE-2018-14734.yml

```diff
@@ -280,8 +294,10 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The feature that this vulnerability affected uses inter-process communication since it
+    involves the RDMA and networking.
```

Comment:
write out Remote Direct Memory Access and consider explaining it briefly, since it's not a common term